service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/controller/utils/ReportParserUtil.java

```diff
@@ -0,0 +1,40 @@
+package uk.nhs.adaptors.oneoneone.cda.report.controller.utils;
+
+import java.io.StringReader;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.dom4j.Document;
+import org.dom4j.DocumentException;
+import org.dom4j.Node;
+import org.dom4j.io.SAXReader;
+
+public class ReportParserUtil {
+
+    public enum ReportElement{
```

Comment:
I wouldn't make it an inner class. Can we extract it? Also, please remember about formatting. There's a missing space after the class name.